### DISCUSSION TITLE
ERROR    Uncaught exception: path::operator/(): expected a relative path!

### DISCUSSION THREAD

**Discussion by jwetalley1** [NVlabs/instant-ngp#648] [Q&A]
I think I am specifying the network path incorrectly.  Your help would be greatly appreciated. Thanks. 

(ngp) C:\Tutorial\ngp\instant-ngp\build>testbed --scene C:\Tutorial\ngp\instant-ngp\data\nerf\fox --network C:\Tutorial\ngp\instant-ngp\configs\nerf\fox\
16:42:50 INFO     Loading NeRF dataset from
16:42:50 INFO       C:\Tutorial\ngp\instant-ngp\data\nerf\fox\transforms.json
16:42:50 SUCCESS  Loaded 50 images after 0s
16:42:50 INFO       cam_aabb=[min=[1.0229,-1.33309,-0.378748], max=[2.46175,1.00721,1.41295]]
16:42:51 ERROR    Uncaught exception: path::operator/(): expected a relative path!

**Comment by Tom94** [comment]
You'll want to pass a `.json` config to the `--network` argument. (The default would be in `instant-ngp\configs\nerf\base.json`)

**Reply by jwetalley1** [reply]
Thanks for the response.  I think the true issue is the path variable set-up to recognize my root location, which is C:\Tutorial\ngp\instant-ngp.  I am now getting the error below after your suggestion.

(ngp) C:\Tutorial\ngp\instant-ngp\build>testbed --scene C:\Tutorial\ngp\instant-ngp\data\nerf\fox --network instant-ngp\configs\nerf\fox\
12:14:41 INFO     Loading NeRF dataset from
12:14:41 INFO       C:\Tutorial\ngp\instant-ngp\data\nerf\fox\transforms.json
12:14:41 SUCCESS  Loaded 50 images after 0s
12:14:41 INFO       cam_aabb=[min=[1.0229,-1.33309,-0.378748], max=[2.46175,1.00721,1.41295]]
12:14:42 ERROR    Network config path instant-ngp\configs\nerf\fox does not exist.

**Reply by jwetalley1** [reply]
How can I set-up the path variable to globally recognize my root location, which is  C:\Tutorial\ngp\instant-ngp?

I am still getting the following error:
12:14:42 ERROR Network config path instant-ngp\configs\nerf\fox does not exist.

**Reply by gvbgeomatics** [reply]
I am also facing similar issue:

C:\instant-ngp\instant-ngp\build>testbed.exe --scene "C:\instant-ngp\instant-ngp\data\nerf\fox"
19:30:43 INFO     Loading NeRF dataset from
19:30:43 INFO       C:\instant-ngp\instant-ngp\data\nerf\fox\transforms.json
19:30:44 SUCCESS  Loaded 50 images after 0s
19:30:44 INFO       cam_aabb=[min=[1.0229,-1.33309,-0.378748], max=[2.46175,1.00721,1.41295]]
19:30:44 ERROR    Network config path configs\nerf\base.json does not exist.

Then I added the network path to the command but still it is throwing a new exception:
C:\instant-ngp\instant-ngp\build>testbed.exe --scene "C:\instant-ngp\instant-ngp\data\nerf\fox" --network "C:\instant-ngp\instant-ngp\configs\nerf\base.json"
19:31:21 INFO     Loading NeRF dataset from
19:31:21 INFO       C:\instant-ngp\instant-ngp\data\nerf\fox\transforms.json
19:31:21 SUCCESS  Loaded 50 images after 0s
19:31:21 INFO       cam_aabb=[min=[1.0229,-1.33309,-0.378748], max=[2.46175,1.00721,1.41295]]
19:31:21 ERROR    Uncaught exception: path::operator/(): expected a relative path!

I am not sure what's going wrong here. Any ideas?

**Reply by codgas** [reply]
`--netowrk ./instant-ngp/configs/nerf/base.json` passing it as a relative path like that fixed the problem for me

**Reply by gvbgeomatics** [reply]
Yes. Relative parth is required. We fixed it and enabled NVIDIA Instant NeRF on Cloud with the choice of GPU configurations and storage options for general public use. More details here: https://nebulacloud.ai/connect/blogs/how-to-use-nvidia-instant-nerf-on-nebula-cloud
